README.md

```diff
@@ -64,6 +64,14 @@ By default, this plugin only compiles ["secure" schemas][secure]. This can be di
 
 By default, only files with a `.schema` extension are compiled. If you have different requirements you can pass a Regexp to `filter` for the plugin to use.
 
+### `formats`
```

Comment:
Would it make sense to call this `addFormats` as a. this aligns better with the ajv documentation and b. `formats` sounds a little like it could be a list of things?